fuel-core/Cargo.toml

```diff
@@ -28,12 +29,14 @@ env_logger = "0.9"
 fuel-asm = { version = "0.1", features = ["serde-types"] }
 fuel-storage = { version = "0.1" }
 fuel-tx = { version = "0.2", features = ["serde-types"] }
+fuel-txpool = { path = "../txpool", version = "0.2" }
```

Comment:
Should use an exact version for local dependencies, to avoid headaches if we ever mess up semver. And makes find-and-replace easier.